test/support/stub_const.rb

```diff
@@ -0,0 +1,11 @@
+# typed: true
+# frozen_string_literal: true
+
+module StubConst
+  def with_stubbed_const(klass, const, val)
```

Comment:
You're going to need a shim for minitest/test to give Sorbet a hint that this method exists.

`in sorbet/rbi/shims/minitest/test.rb`
```
# typed: true

class Minitest::Test
  include StubConst
end
```